oaknational/Oak-Web-Application
# feat: skip button for lesson overview - LESQ-761

## Description

Music year: 1966

- Adds skip button to lesson overview item container (video / slide / worksheet)
- Button is only visible when focused
- Button takes you to the next item (resource) on the page


## Issue(s)



## How to test

1. Go to https://deploy-preview-2558--oak-web-application.netlify.thenational.academy
Tab through lesson overview page, you should see skip button appearing when focused, pressing button should skip content and take you to the the next item.



## Checklist
